personalities/ct-static-api/ct_server_gcp/main.go

```diff
@@ -280,9 +276,9 @@ func setupAndRegister(ctx context.Context, deadline time.Duration, cfg *configpb
 		opts.CertificateQuotaUser = ctfe.QuotaUserForCert
```

Comment:
Can quota stuff come out too?

Reply:
I tried to remove it a few minutes ago, but I'll get back to this when we figure out what to do for quota, this might still be useful. I've added an AI on #88.